src/hotspot/cpu/ppc/gc/shenandoah/shenandoahBarrierSetAssembler_ppc.cpp

```diff
@@ -1,6 +1,6 @@
 /*
  * Copyright (c) 2018, 2021, Red Hat, Inc. All rights reserved.
- * Copyright (c) 2012, 2021 SAP SE. All rights reserved.
+ * Copyright (c) 2012, 2022 SAP SE. All rights reserved.
  * Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
```

Comment:
I believe line 4 should deleted; the copyright header change here is unnecessary.

Reply:
will remove.  I noticed that amazon had also contributed to this file, but changes were very minor.